src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/reporting/model/PlacementApplicationReportRow.kt

```diff
@@ -30,6 +31,7 @@ data class PlacementApplicationReportRow(
   val applicationWithdrawalDate: LocalDate?,
   val applicationWithdrawalReason: String?,
   val bookingID: String?,
+  val bookingMadeDate: LocalDateTime?,
```

Comment:
I think we've maybe got a mismatch between the name (date) and the format (timestamp)?

perhaps:
- If the report requires a date then we need to convert the timestamp to date
- if the report requires a timestamp then we need to name the field/property/column accordingly e.g. `bookingMadeAt` 

We've tried to stick to the naming convention of:

- `...At` for a timestamp, e.g. `bookingMadeAt`
- `..On` for a date, e.g. `bookingMadeOn`

Reply:
Good spot! looking at the existing report columns we are only returning a LocalDate (not LocalDateTime), so i'm going to change these occurrences to Date only.

As for naming, for these two reports we're using the 'Date' suffix everywhere LocalDate is used, so for consistency i'll stick with this naming

Reply:
👍 local conventions trump!